queries.sql

```diff
@@ -1,3 +1,11 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name LIKE '%mon';
+SELECT name FROM animals where date_of_birth BETWEEN DATE '2016-01-01' AND '2019-01-01';
+SELECT name, neutered, escape_attempts FROM  animals WHERE neutered = 'yes' AND escape_attempts < 3;
+SELECT name FROM animals WHERE name = 'Agumon' OR name = 'Pikachu';
```

Comment:
- [ ] This query is about selecting the date of birth not names of all animals named either "Agumon" or "Pikachu" please fix that

Reply:
This one will show only names in this query!